fulp_modules/batong/code/batongcode.dm

```diff
@@ -0,0 +1,11 @@
+
+
+/obj/item/toy/plush/batong
+	name = "batong"
+	desc = "A cheaply made toy. looks like it need some recharge"
+	icon = 'fulp_modules/batong/icon/toybatong.dmi'
+	icon_state = "toy"
+	inhand_icon_state = "toy"
+	attack_verb_continuous = list("stung")
```

Comment:
stung?

Reply:
like stun but with a g, like baton batong. but problably i'm going to change that text and the description. i was very tired when i wrote those lines.